lib/view_model/active_record/update_context.rb

```diff
@@ -165,6 +165,10 @@ def run!(deserialize_context:)
 
       @release_pool.release_all!
 
+      if updated_viewmodels.present?
```

Comment:
Confirm my understanding: it's only necessary to run when some model is changed, even in the presence of deferred foreign key constraints, because there's no way to delete a record without changing or deleting the record the constraint applies to.